.template/lib/template/errors.rb

```diff
@@ -1 +1,3 @@
+# rubocop:todo Style/ClassAndModuleChildren
```

Comment:
I'll fix this when working on refactoring the template. Because it needs to load the template module in prior. 🙏